tests/fixups/TraceFixup/DynamicLinkLibraryFixup.cpp

```diff
@@ -41,12 +68,39 @@ auto LoadLibraryImpl = psf::detoured_string_function(&::LoadLibraryA, &::LoadLib
 template <typename CharT>
 HMODULE __stdcall LoadLibraryFixup(_In_ const CharT* libFileName)
 {
+	LARGE_INTEGER TickStart, TickEnd;
+	QueryPerformanceCounter(&TickStart);
     auto entry = LogFunctionEntry();
     auto result = LoadLibraryImpl(libFileName);
+QueryPerformanceCounter(&TickEnd);

```

Comment:
indentation

Reply:
Addressed in commit 727a9e9856d361f686523f2ea2907876c3ef9b23 on 11/28